internal/genny/docker/docker.go

```diff
@@ -19,19 +24,51 @@ func New(opts *Options) (*genny.Generator, error) {
 		"opts": opts,
 	}
 
-	g.Box(packr.New("github.com/gobuffalo/buffalo/common", "../docker/templates/common"))
+	common, err := fs.Sub(templates, "templates/common")
+	if err != nil {
+		return g, err
+	}
+
+	if err := g.FS(common); err != nil {
+		return g, err
+	}
 
 	switch opts.Style {
 	case "multi":
-		g.Box(packr.New("github.com/gobuffalo/buffalo/multi", "../docker/templates/multi"))
+		multi, err := fs.Sub(templates, "templates/multi")
```

Comment:
There is a follow up action from this one: moving to only have multi-stage docker file. That would simplify this generator.

Reply:
Yes I've seen https://github.com/gobuffalo/cli/issues/42 but we should resolve that task in a dedicated PR. I didn't want to do too much at once.

Reply:
Yes, I'm on that one.